robotsim.py

```diff
@@ -257,20 +353,26 @@ def getColor(self):
         return 'white'
 
     def sendMessageRescueBase(self, coordinate, path = None):
-        row = coordinate.x
-        col = coordinate.y
+        row = coordinate.y
+        col = coordinate.x
         if map.is_valid_coordinate(row, col) and map.tiles[row][col].envType == "people":
-            map.tiles[row][col].envType = "blank"
+            map.tiles[row][col].envType = "clear"
+            map.tiles[row][col].color = "white"
```

Comment:
generate_map() should be called after editing the map.

Reply:
buena esa